lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/IMRUBroadcastReduceTest.cs

```diff
@@ -56,22 +57,37 @@ void TestIMRUBroadcastReduceWithFTOnLocalRuntime()
             CleanUp(testFolder);
         }
 
+        [Fact]
+        [Trait("Description", "Run IMRU broadcast and reduce example with random failures as test.")]
+        void TestImruBroadcastReduceWithRandFailuresOnLocalRuntime()
+        {
+            var testFolder = DefaultRuntimeFolder + TestId;
+            TestIMRUBroadcastReduce(false, true, false, testFolder);
+            var lines = ReadLogFile(DriverStdout, "driver", testFolder, 240);
+            var completedEvaluatorCount = GetMessageCount(lines, "Received CompletedEvaluator");
+            Assert.Equal(NumNodes, completedEvaluatorCount);
+            CleanUp(testFolder);
```

Comment:
Do you want to add a check that overall job has completed successfully? Similar to what we do in `TestFailedMapperOnLocalRuntime` (we don't do it in other tests in this file, but we should)